### PR TITLE
add extra.opts to saveGIF

### PR DESCRIPTION
**Pull request by stla**
The `saveGIF` function now has the argument `extra.opts` which is passed to `im.convert`. An example is given: `inst/examples/spinner-ex.R`.

### CODE REVIEW

File: inst/examples/spinner-ex.R [hunk 1]
```diff
@@ -0,0 +1,27 @@
+library(scales) # for the alpha() function
```
Comment (yihui) [inline inst/examples/spinner-ex.R]: It is not worth introducing this new dependency only for an example (and only one function in **scales** is used).

File: R/saveGIF.R [hunk 2]
```diff
@@ -59,7 +60,7 @@
 #' @export
 saveGIF = function(
   expr, movie.name = 'animation.gif', img.name = 'Rplot', convert = 'convert',
-  cmd.fun, clean = TRUE, ...
+  cmd.fun, clean = TRUE, extra.opts="", ...
```
Comment (yihui) [inline R/saveGIF.R]: Spaces around operators, such as `=`.

**Comment by stla** [comment]
Hi Yihui,
 I'm lost with pull requests ^^
Did you get my revision ? (commit "revision #93"). I've done the changes you requested.

**Comment by yulijia** [comment]
@yihui @stla  would you mind we merge it then edit?

**Pull request merged**